testing/component-testing-rtl/src/__tests__/1-using-react-testing-library.test.js

```diff
@@ -109,14 +112,14 @@ already tested the open and close functionality of the
 Dialog, the IconButton, and the Icon individually,
 we only need to test what's unique about this particular component
 
-🛠️ Write a test for 3 things
+🛠️ Write a test for 2 things
```

Comment:
Can you spell out `two` here, as you've done elsewhere in this exercise?

Reply:
I ended up rewording these instructions a little bit...